### PR TITLE
docs: Update frontend configs to latest versions

### PR DESCRIPTION
**Pull request by FabianLars**
_still not reading the template_ 🕺

### CODE REVIEW

File: src/content/docs/start/Frontend Configuration/index.mdx [hunk 1]
```diff
@@ -28,18 +22,24 @@ Below is a checklist of common scenarios needed to integrate a frontend with Tau
...
 {/* TODO: Concept of how to do a client-server relationship? */}
 
-- Use static site generation (SSG). Tauri doesn't officially support server based alternatives (such as SSR).
+- Use static site generation (SSG), single-page applications (SPA), or classic multi-page apps (MPA). Tauri does not natively support server based alternatives (such as SSR).
 - For mobile development, a development server of some kind is necessary that can host the frontend on your internal IP.
 - Use a proper client-server relationship between your app and your API's (no hybrid solutions with SSR).
 
 ## JavaScript
 
+{/* TODO: Help me with the wording here lol */}
```
Comment (FabianLars) [inline src/content/docs/start/Frontend Configuration/index.mdx:31]: 😢

File: src/content/docs/start/Frontend Configuration/index.mdx [hunk 2]
```diff
@@ -28,18 +22,24 @@ Below is a checklist of common scenarios needed to integrate a frontend with Tau
...
 	<LinkCard title="Next.js" href="/start/frontend-configuration/nextjs" />
 	<LinkCard title="Nuxt" href="/start/frontend-configuration/nuxt" />
 	<LinkCard title="Qwik" href="/start/frontend-configuration/qwik" />
-	<LinkCard title="Svelte" href="/start/frontend-configuration/svelte" />
-	<LinkCard title="Vite" href="/start/frontend-configuration/vite" />
+	<LinkCard title="SvelteKit" href="/start/frontend-configuration/sveltekit" />
+	<LinkCard
+		title="Vite (recommended)"
+		href="/start/frontend-configuration/vite"
+	/>
```
Comment (vasfvitor) [inline src/content/docs/start/Frontend Configuration/index.mdx:42]: Since Vite is the recommended, I think placing it in first position makes sense

Comment (FabianLars) [reply src/content/docs/start/Frontend Configuration/index.mdx:42]: probably a good idea if we all agree to mark it as recommended. Just have to remember that the rest keeps being sorted alphabetically so we don't end up with an unsorted mess 😂

Comment (vasfvitor) [reply src/content/docs/start/Frontend Configuration/index.mdx:42]: well, CTA always will create a vite project if ts/js is chosen right? that's beyond recommended I'd say 😄. I personally agree in having it recommended

>Just have to remember that the rest keeps being sorted alphabetically so we don't end up with an unsorted mess

Oh, so... I was thinking about smth that fits this. Having link guide for vite separated in a section before those guides, but this is stuff for another PR. I think for now then let's no change its position then.

**Review by vasfvitor — APPROVED** [review]
In my opinion we can leave the wording to a follow up PR, considering there's more context to add. @dreyfus92 do you have any suggestion here?

**Review by dreyfus92 — APPROVED** [review]
This looks good to me, we can definitely save the JS section for later.

**Pull request merged**